guides/release/components/component-state-and-actions.md

```diff
@@ -418,4 +418,10 @@ export default class DoubleItComponent extends Component {
 }
 ```
 
+## Learn more
+
+As components are frequently used, patterns are very important to help reduce a lot of bugs and unforeseen issues. More in-depth guide can be found in the following chapter [Patterns for components.](../../in-depth-topics/patterns-for-components/)
```

Comment:
The meaning of the word "as" wasn't clear to me. I think you meant to use the word to mean "since"?

To remove ambiguity, what do you think of:

```suggestion
You will frequently create components in an app. Establishing patterns early can help reduce bugs and unforeseen issues. Learn more from the chapter [Patterns for Components](../../in-depth-topics/patterns-for-components/).
```

I removed "very" and "a lot of" so that the sentence may sound neutral.